numpy-dev-xfails.txt

```diff
@@ -0,0 +1,61 @@
+# asarray(copy=False) is not yet implemented
+array_api_tests/test_creation_functions.py::test_asarray_arrays
+
+# finfo(float32).eps returns float32 but should return float
+array_api_tests/test_data_type_functions.py::test_finfo[float32]
+
+# Array methods and attributes not already on np.ndarray cannot be wrapped
+array_api_tests/test_has_names.py::test_has_names[array_method-to_device]
+array_api_tests/test_has_names.py::test_has_names[array_attribute-device]
+
+# linalg tests require https://github.com/data-apis/array-api-tests/pull/101
+# cleanups. Also some tests are using .mT
+array_api_tests/test_linalg.py::test_solve
+
+# Array methods and attributes not already on np.ndarray cannot be wrapped
+array_api_tests/test_signatures.py::test_array_method_signature[__array_namespace__]
```

Comment:
This method exists already for a few weeks, and I just merged what's probably the fix to make this test pass: https://github.com/numpy/numpy/pull/25595